src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  box-sizing: border-box;
+  display: flex;
+  width: 100px;
+  height: 96px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 96px;
+  padding-right: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+div.is_active {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
1. do not use selector by tag,
2. you should use `nth-child` pseudo class here to set active stars

Reply:
Fixed inaccuracies in the code, used pseudo-classes nth-child for active stars